pytest_fixtures/component/provisioning_template.py

```diff
@@ -1,7 +1,12 @@
 # Provisioning Template Fixtures
 import pytest
+from box import Box
 from nailgun import entities
+from packaging.version import Version
 
+from robottelo import constants
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.api.utils import wait_for_tasks
```

Comment:
Do not import these two, use the Host object instead.
See https://github.com/SatelliteQE/robottelo/blob/0e441a0998c7a3208edeb7a843bdda9af4995ee5/tests/foreman/api/test_remoteexecution.py#L64

Reply:
Thanks, Updated.